### PR TITLE
Remove unnecessary line breaks from Onboarding pages

### PR DESCRIPTION
**Pull request by johnny9**
https://github.com/bitcoin-core/gui-qml/issues/282

[Windows](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip?branch=pull/283)
[Intel macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip?branch=pull/283)
[Apple Silicon macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/insecure_mac_arm64_gui.zip?branch=pull/283)
[ARM64 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip?branch=pull/283)

### CODE REVIEW

**Comment by johnny9** [comment]
| main | PR |
|--|--|
|Screenshot from 2023-03-07 08-43-37 |Screenshot from 2023-03-07 08-39-08 |
|--|--|
|Screenshot from 2023-03-07 08-43-44 |Screenshot from 2023-03-07 08-39-19 |

File: src/qml/pages/onboarding/OnboardingBlockclock.qml [hunk 1]
```diff
@@ -20,6 +20,8 @@ InformationPage {
     }
     bold: true
     headerText: qsTr("The block clock")
-    description: qsTr("The Bitcoin network targets a new block every\n10 minutes. Sometimes it's faster and sometimes slower.\n\nThe block clock indicates each block on a dial\nthat represents the current day.")
+    description: qsTr("The Bitcoin network targets a new block every 10 minutes. " +
+        "Sometimes it's faster and sometimes slower. The block clock indicates each " +
+        "block on a dial that represents the current day.")
```
Comment (jarolrod) [inline src/qml/pages/onboarding/OnboardingBlockclock.qml:25]: The design still specifies that there is atleast one line of text in the onboarding pages text that is on a separate line: https://www.figma.com/file/ek8w3n3upbluw5UL2lGhRx/Bitcoin-Core-App-Design?node-id=5952%3A389674&t=yj4lHUbV7gL4ByjQ-4

Comment (jarolrod) [reply src/qml/pages/onboarding/OnboardingBlockclock.qml:25]: You can add in a subtext here in the information page control, this is aliased to the subtext of the header control.

Comment (johnny9) [reply src/qml/pages/onboarding/OnboardingBlockclock.qml:25]: ill try the subtext out

Comment (johnny9) [reply src/qml/pages/onboarding/OnboardingBlockclock.qml:25]: Ill look at subtext, but I actually just removed too many line breaks. Ill update

**Comment by johnny9** [comment]
Update from[ 94ce729 ](https://github.com/bitcoin-core/gui-qml/commit/94ce729f35a1b18b803aaa39a0b0e3f1ff67791a)to[ 656df9](https://github.com/bitcoin-core/gui-qml/commit/656df955a036133bfbd26ee5037bb598c8d21aa1)
 - Re-add the linebreaks that we want

| main | PR |
|--|--|
|Screenshot from 2023-03-07 08-43-37 |Screenshot from 2023-03-07 08-43-37|
|Screenshot from 2023-03-07 08-43-44 | Screenshot from 2023-03-07 08-43-44|

(ignore "normal" at the topright, have an opengl overlay on testing gsync)

**Comment by johnny9** [comment]
Update from[ 656df95 ](https://github.com/bitcoin-core/gui-qml/commit/656df955a036133bfbd26ee5037bb598c8d21aa1)to 1ef15eb:
  - Two \n so MacOs builds display the break correctly

**Review by jarolrod — APPROVED** [review]
ACK 1ef15eb2c917627602685f048288c018b484189c

**Pull request merged**